scripts/_common.sh

```diff
@@ -8,7 +8,7 @@
 nodejs_version=16
 
 # Terraforming-mars commit
-TM_version="38c69c7"
+TM_version="78e5e5d"
```

Comment:
(btw it would be great it we could use `[resources.sources]` and `ynh_setup_source` rather than git clone which is uuh not the standard :sweat_smile:)

Reply:
Oh I didn't see your comment!

Yes I agree with you. I first tried with `ynh_setup_source`, but I had git errors so I tried with the "official" installation instructions.
It was a good way to learn some git things because I'm still a great noob 🤓

But I plan to try again with `ynh_setup_source` when I have some time.